### DISCUSSION TITLE
How to prevent CSS chunks and simply write it in style tag inside Head.

### DISCUSSION THREAD

**Discussion by Croodo** [nuxt/nuxt#18985] [Questions]
Nuxt3 by default makes CSS chunks. I want to use my CSS inside the `<style></style> ` tag in the head.
What is the configuration for that? 
I'm Using TailwindCSS.

**Comment by ghost** [comment]
Hey, did you manage to find the answer to this @Croodo ?

**Reply by AloisSeckar** [reply]
I didn't actually try it yet, but I would try to expermient with [useHead](https://nuxt.com/docs/api/composables/use-head) as it has `style` option, which I think would do the thing.

Alternatively, it is [possible to link entire external stylesheet](https://nuxt.com/docs/getting-started/styling#external-stylesheets).

**Reply by ghost** [reply]
Well this is one probable solution: https://vitejs.dev/config/build-options.html#build-csscodesplit
